package.json

```diff
@@ -4,34 +4,33 @@
 	"description": "Typescript Community Bot",
 	"main": "dist/index.js",
 	"dependencies": {
-		"@typescript/twoslash": "^2.1.0",
-		"algoliasearch": "^4.8.6",
-		"cookiecord": "^0.8.18",
+		"@typescript/twoslash": "^3.2.1",
+		"algoliasearch": "^4.14.2",
+		"discord.js": "^14.6.0",
 		"dotenv-safe": "^8.2.0",
-		"html-entities": "^2.3.2",
+		"html-entities": "^2.3.3",
 		"lz-string": "^1.4.4",
-		"node-fetch": "^2.6.7",
-		"npm-registry-fetch": "^9.0.0",
-		"parse-duration": "^0.4.4",
-		"pg": "^8.3.0",
-		"prettier": "^2.2.1",
-		"pretty-ms": "^7.0.0",
-		"tar": "^6.1.0",
-		"typeorm": "^0.2.25"
+		"npm-registry-fetch": "^14.0.2",
+		"parse-duration": "^1.0.2",
+		"pg": "^8.8.0",
+		"prettier": "^2.7.1",
+		"pretty-ms": "^8.0.0",
+		"tar": "^6.1.12",
+		"typeorm": "^0.3.10",
+		"undici": "^5.12.0"
```

Comment:
node-fetch -> undici because node-fetch requires ESM, and I didn't want that pain today.